minject/__init__.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, api, path): ...
 something = registry['something_i_need']
 """
 
-__version__ = "1.0.0"
+__version__ = "1.1.0"
```

Comment:
Can we use a pre-release version here, like `1.1.0-beta.1`? I'm reluctant to go straight to a release version in an open-source project, because if we discover API issues that need to be fixed that would entail a breaking change.

(For example, I wonder if it would be cleaner to split `@async_context` into orthogonal `@async` and `@context` decorators, but if we've already released `@async_context` we ought to keep supporting that too.)